### PR TITLE
[CDAP-14341] make metadata optional and add license info in market

### PR DESCRIPTION
**Pull request by elfenheart**
JIRA: https://issues.cask.co/browse/CDAP-14341
Build: https://builds.cask.co/browse/CDAP-URUT97

### CODE REVIEW

File: cdap-ui/app/cdap/components/MarketPlaceUsecaseEntity/LicenseRow.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+  if (licenseInfo.url) {
+    tagProps.href = licenseInfo.url;
+    tagProps.target = '_blank';
+  }
+
+  return (
+    <React.Fragment>
+      <div>
+        {T.translate('features.MarketPlaceEntity.Metadata.licenseInfo')}
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/MarketPlaceUsecaseEntity/LicenseRow.tsx:46]: Is it possible to have a wrapper div that is display flex and have it accept a prop for layout (direction - stacked vs side by side) and a prop for the label? Seems like for the most part it is the same component as the one in MarketPlaceEntity.

File: cdap-ui/app/cdap/components/MarketPlaceEntity/LicenseRow.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+  if (licenseInfo.url) {
+    tagProps.href = licenseInfo.url;
+    tagProps.target = '_blank';
+  }
+
+  return (
+    <div>
+      <span>
+        <strong> {T.translate('features.MarketPlaceEntity.Metadata.licenseInfo')} </strong>
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/MarketPlaceEntity/LicenseRow.tsx:46]: Any specific reason why its strong in solutions and not in the other places?

**Review by ajainarayanan — COMMENTED** [review]
Minor comments.

**Comment by ajainarayanan** [comment]
@elfenheart The JIRA also talks about removing the beta tags and removing stream upload steps in datapacks. Is it coming in a separate PR or did we decide something else?

File: cdap-ui/app/cdap/components/MarketPlaceEntity/LicenseRow.tsx (hunk 2 above)
Comment (elfenheart) [reply cdap-ui/app/cdap/components/MarketPlaceEntity/LicenseRow.tsx:46]: the design for solutions is different vs everywhere else.

**Comment by elfenheart** [comment]
@ajainarayanan the beta tags and removing stream upload is just configuration from market. We don't hardcode that in the UI.

File: cdap-ui/app/cdap/components/MarketPlaceEntity/LicenseRow.tsx (hunk 2 above)
Comment (ajainarayanan) [reply cdap-ui/app/cdap/components/MarketPlaceEntity/LicenseRow.tsx:46]: 🤦‍♂️ 👍

**Review by ajainarayanan — APPROVED** [review]
👍

**Pull request merged**